src/output-json-frame.c

```diff
@@ -153,7 +153,8 @@ static void FrameAddPayloadTCP(JsonBuilder *js, const TcpStream *stream, const F
     }
 
     if (frame->len >= 0) {
-        sb_data_len = MIN(frame->len, (int32_t)sb_data_len);
+        DEBUG_VALIDATE_BUG_ON(frame->len > UINT32_MAX);
```

Comment:
Frame's `len` is defined to be `int64_t`, why should it be a debug condition if it goes beyond `UINT32_MAX`?

Reply:
You are right, we should use a uint64_t for this MIN, and hence use a new u64 variable as sb_data_len is u32 per `StreamingBufferGetData`